templates/projects/tasks_browse.html

```diff
@@ -244,34 +251,38 @@
                     {% endif %}
                 {% endif %}
                 {% if 'created' in filter_data.display_columns %}
-                <td>{{ t.created }}</td>
+                <td><div style="width: 100px;">{{ t.created }}</div></td>
                 {% endif %}
                 {% if 'finish_time' in filter_data.display_columns %}
-                <td>{{ t.finish_time or '' }}</td>
+                <td><div style="width: 100px;">{{ t.finish_time or '' }}</div></td>
                 {% endif %}
                 {% if 'gold_task' in filter_data.display_columns %}
                     {% if t.calibration %}
-                        <td style="width: 100%">
-                            <!-- Allow project owner, sub-admin co-owners, and admins to update Gold tasks -->
+                        <td>
+                            <div class="make-gold-cell">
+                                <!-- Allow project owner, sub-admin co-owners, and admins to update Gold tasks -->
+                                {% if (current_user.subadmin and current_user.id in project.owners_ids) or current_user.admin %}
+                                    <a class="gold-task label label-warning" target="_blank" href="{{ url_for('project.task_presenter', short_name=project.short_name, task_id=t.id, mode='gold') }}">Update</a>
+                                {% else %}
+                                    <div class="gold-task label label-default" title="Only Project owners can update Gold tasks.">Update</div>
+                                {% endif %}
+                            </div>
+                        </td>
+                    {% else %}
+                    <td>
+                        <div class="make-gold-cell">
+                            <!-- Allow project owner, sub-admin co-owners, and admins to make Gold tasks -->
                             {% if (current_user.subadmin and current_user.id in project.owners_ids) or current_user.admin %}
-                                <a class="gold-task label label-warning" target="_blank" href="{{ url_for('project.task_presenter', short_name=project.short_name, task_id=t.id, mode='gold') }}">Update</a>
+                                <a class="gold-task label label-info" target="_blank" href="{{ url_for('project.task_presenter', short_name=project.short_name, task_id=t.id, mode='gold') }}">Make Gold</a>
                             {% else %}
-                                <div class="gold-task label label-default" title="Only Project owners can update Gold tasks.">Update</div>
+                                <div class="gold-task label label-default" title="Only project owners can make Gold tasks.">Make Gold</div>
```

Comment:
Will this message present to regular user when updating a gold task that will be visible to them like a regular task?

Reply:
Hm, I don't believe this logic has changed from the prior version. It may be displaying here due to the newly added `<div class="make-gold-cell">`, which offsets the code.

Reply:
This need to revisit when task browse would be available for regular user, may be under a different ticket as this ticket is related to collapsible nav bar. Thanks!